bedrock/mozorg/templates/mozorg/contribute/contribute-2020.html

```diff
@@ -0,0 +1,252 @@
+            <h3 class="contribute-mission-donate-icon">Donate</h3>
+            <p>Help fund our mission</p>
+          </a>
+        </li>
+
+        <li class="contribute-mission-card">
+          <a href="https://twitter.com/mozilla">
+            <h3 class="contribute-mission-follow-icon">Follow Mozilla</h3>
+            <p>News and volunteer opportunities</p>
+          </a>
+        </li>
+      </div>
+    </div>
+  </section>
+
+  <div class="mzp-l-content contribute-feature-container">
+    <section class="contribute-feature">
+
+      <div class="mzp-c-card-feature-media contribute-feature-media-wrapper">
+        <div class="contribute-feature-media">
+          <img src="{{ static('img/contribute/contribute-madeby-feature.jpg') }}" alt="">
+        </div>
+      </div>
+
+      <div class="contribute-feature-content">
+        <div class="contribute-feature-content-container">
+          <h2>Made by passionate people like you</h2>
+          <div class="contribute-feature-desc">
+            <p>When you contribute to Mozilla, you become part of a worldwide
+              community made up of people from around the globe who believe
+              that we all have a role in ensuring that the internet remains a force
+              for good.</p>
+
+            <p>We believe community collaboration is vital to
+              creating an internet that is diverse, innovative, and
+              accountable to the people who need it most. Whether you have a background in
+              tech, community organizing, or just a computer and some free
+              time, you can help us make the internet a better place.</p>
+          </div>
+        </div>
+      </div>
+    </section>
+  </div>
+
+  <section class="mzp-l-content">
+    <aside class="mzp-c-newsletter">
+      <div class="mzp-c-newsletter-image">
+        {{ high_res_img('img/contribute/newsletter.png', {'alt': ''}) }}
+      </div>
+
+      <div class="newsletter-content">
+        {{ email_newsletter_form(
+          title=_('Not sure yet?'),
+          subtitle=_('Get community and contribution news to your inbox'),
+          desc=_('Subscribe to our newsletter, join Mozillians all around the world, and learn about impactful opportunities to support Mozilla’s mission.'),
+          newsletters='about-mozilla',
+          protocol_component=True,
+          include_language=False)
+        }}
+      </div>
+    </aside>
+  </section>
+
+  <section class="contribute-banner-wrapper">
+    <div class="contribute-banner contribute-banner-gethelp">
+      <div class="contribute-banner-content">
+        <h2 class="contribute-banner-gethelp-title">Get Help</h2>
+        <p>Mozilla has a dedicated team of volunteers to help with whatever
+          issues you may be experiencing.</p>
+      </div>
+      <div class="contribute-banner-cta">
+        <a href="https://support.mozilla.org/{{ utm_params }}" class="mzp-c-button mzp-t-product">
```

Comment:
I don't know what the rules are here as things keep changing, but shouldn't `mzp-t-product` be reserved for actions such as downloading or signing up for a product/service? Or is this now applicable to _anything_ product related?

Reply:
I don't know the latest rules either. Buttons are changing in the next release of Protocol and we need to establish clearer guidelines on when to use the blue product button. I'm inclined to just let it slide for now.